radial-gauge/viz-gauge.js

```diff
@@ -184,6 +184,12 @@ function processData(data, queryResponse, config, viz) {
   }
   var mesID = meas[0]["name"];
   var mesData = data[0][mesID];
+
+  // if value/rendered aren't there then this is an object of objects (pivot)
+  if(mesData["value"] === undefined && mesData["rendered"] === undefined) {
```

Comment:
fwiw, we tend to use the `in` check for this.

```suggestion
  if (!('value' in mesData && ! 'rendered' in mesData)) {
```